src/cloud/configure/setup-b2b.md

```diff
@@ -1,84 +1,128 @@
+   ```
 
-{% include cloud/cli-get-started.md %}
+1. Update the project dependencies.
 
-## Add B2B in the cloud {#add}
+   ```bash
+   composer update
+   ```
 
-You need to add the module to `composer.json`. All extensions and modules must be added to this file. These instructions are specific to {{site.data.var.ece}}. For more information, you can also review the [installation instructions]({{ site.baseurl }}/extensions/b2b/) in the B2B guide.
+1. Use `ece-tools` to refresh the configuration and enable the B2B module.
+
+   ```bash
+   ./vendor/bin/ece-tools module:refresh
+   ```
 
-1. Open a terminal application.
-1. Change to your local development environment root directory.
-1. Install the B2B module using composer.
+1. Complete any upgrade for the Magento application and the B2B module.
 
    ```bash
-   composer require magento/extension-b2b
+   bin/magento setup:upgrade
    ```
 
-   You may be prompted to enter your Magento Authentication keys (public and private). If copying and pasting your keys, do not introduce additional spaces. Spaces could cause the following error:
+1. Clean the cache.
 
-   ```terminal
-   InvalidArgumentException - Could not find package magento/extension-b2b at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability.
+   ```bash
+   bin/magento cache:clean
    ```
 
-1. Add and commit the updated `composer.json` and `composer.lock` files.
+1. Add, commit, and push code changes.
 
    ```bash
-   git add composer.json composer.lock && git commit -a -m "install b2b module"
+   git add -A
    ```
 
-1. Enable all missing modules, including B2B, for updating.
+   ```bash
+   git commit -m "install b2b module"
+   ```
 
    ```bash
-   ./vendor/bin/ece-tools module:refresh
+   git push origin <branch-names>
    ```
 
-1. Complete the upgrade with B2B using the following command:
+1. After the build and deploy finishes, use a SSH to log in to the remote environment and verify the B2B module installed.
 
    ```bash
-   php bin/magento setup:upgrade
+   bin/magento module:status Magento_B2b
    ```
 
-If you have a config.php file as part of your deployment, you should also add the B2B module in the modules section of the file.
+   An extension name uses the format: `<VendorName>_<ComponentName>`.
 
-1. Change to the app/etc directory.
-1. Edit the config.php with a text editor.
-1. In the modules list, add the B2B module.
-1. Save the file and update Git.
+   Sample response:
 
-   ```bash
-   git add -f app/etc/config.php
+   ```terminal
+   Module is enabled
    ```
 
-   ```bash
-   git commit -a -m “Add config.php.”
+If you encounter errors, see [extension deployment failure][trouble].
+
+### Custom config.php file
+
+When adding the B2B module, the [`app/etc/config.php`][config] file is automatically updated. However, if you have a custom config file, you must update that manually.
+
+{:.procedure}
+To add the B2B module to a custom `config.php` file:
```

Comment:
They should not edit this file manually

Reply:
So, what do they do if they have a custom config?